config/index.js

```diff
@@ -34,7 +34,8 @@ const config = {
     api: {
       bodySizeLimit: process.env.API_BODY_SIZE_LIMIT,
       parameterLimit: process.env.API_PARAMETER_LIMIT,
-      port: process.env.PORT
+      port: process.env.PORT,
+      randomWeetsApi: process.env.API_RANDOM_WEETS
```

Comment:
This would add a new property inside the common object, something like `geekJokeApi`